### ISSUE TITLE
Feature request: invite others to submap only

### ISSUE THREAD

**Issue by juliusgb** [cdaniel/atlas2#100]
Hi @cdaniel,

Not sure if this is the way to make feature requests, but one I had in mind was to be able to invite people only to a sub-map level. It would be an extension of the current functionality, which invites people on a map (organisation) level.

Besides me wanting it, the gap is also described here[1] on page 1 (question 3) and page 5 (question 2 & 3).

Maybe this functionality is already on your backlog.

[1] - https://leadingedgeforum.com/publication/how-people-use-wardley-maps/

Thanks,
Julius

**Comment by cdaniel** [comment]
@juliusgb,
thanks a lot for this feature request. I have been thinking about it for quite some time, but map-level access has a serious disadvantage - it creates an artificial border between the submap and the rest of the company, and hides components from certain groups of people.
This is essentially against the Wardley Doctrine - as if you are working on a map, you have to have access to all existing components in the company, because only then you can avoid duplication.
That said, I am not saying I will not implement this, I just have no idea how to do it properly. I would be very grateful if you could shed more light on what do you need this functionality for.

**Comment by juliusgb** [comment]
Yep, sounds right - I'd forgotten about Doctrine (i still have much to learn). And old habits die hard :(

The one scenario I had in mind was when asking an external 3rd party to validate the submap in the context of their industry/speciality. The user need that's higher up the value chain shouldn't be of interest, or relevant, to them. Perhaps this is wishful thinking.

Feel free to de-prioritise this feature request. I don't have many solid data points.

**Comment by cdaniel** [comment]
This is actually quite an interesting point, and if I will find a way to do it right, I will implement it. 

For now, I can only advise creating a separate workspace for the 3rd party.

**Comment by juliusgb** [comment]
Hi @cdaniel

Using a different workspace for 3rd parties works well too.

It's probably late, but I'd like to add more context in case I come back to this a few weeks/months from now.

What I had in mind was to make it possible for others to create their own wardley maps, which, when made available, I could consume somehow (api call, etc). 

E.g., One interest of mine is in 3rd party surveys/articles/infographics that show what are the popular tools or technology stacks, such as this Cloud Native Project (https://github.com/cncf/landscape); or this AWS security primer (https://cloudonaut.io/aws-security-primer/).


When I'm creating my map for user need 'x', I can represent the cloud native approach as a submap. Then when I click on the submap, I can see the landscape from the first link, hopefully where each component is placed in the proper phase of evolution. Ofcourse, I'd review it, discuss it with others, & probably change it to suit my situation, but the initial map would have saved me a lot of time & given me ideas on where to look.

Over a period of time, there would be a marketplace for ready-made maps - at least for product/rental and commoditized components. As for pricing, that "Parking Place LTD" example from the mapping online course had good ideas :)

Does this make sense to you or am I over-thinking things again?
